simple_IMSI-catcher.py

```diff
@@ -94,5 +95,34 @@ def find_imsi(x):
 				if p[72:][:2] == '\x08\x29':
 					# if IMSI 2
 					show_imsi(p[73:][:8])
+	"""
```

Comment:
Is it really a good idea to disable a code block with """ instead of removing the code?